File: .sdk/tm/php/utility/MakeContext.php
<?php
declare(strict_types=1);

// WahlumfragenDatenbank SDK utility: make_context

require_once __DIR__ . '/../core/Context.php';

class WahlumfragenDatenbankMakeContext
{
    public static function call(array $ctxmap, ?WahlumfragenDatenbankContext $basectx): WahlumfragenDatenbankContext
    {
        return new WahlumfragenDatenbankContext($ctxmap, $basectx);
    }
}

File: .sdk/tm/py/pkg/utility/make_context.py
# WahlumfragenDatenbank SDK utility: make_context

from projectname_sdk.core.context import WahlumfragenDatenbankContext


def make_context_util(ctxmap, basectx):
    return WahlumfragenDatenbankContext(ctxmap, basectx)
